test_win.cmd

```diff
@@ -1,38 +1,56 @@
-@CLS
-
 @ECHO OFF
-@ECHO ** Run the NUnit tests from the Windows command line **
+
+@ECHO/
+@ECHO ***************************************
+@ECHO ** Run the NUnit tests for GEDKeeper **
+@ECHO ***************************************
 @ECHO/
 
 :: Set up our environment.
 SET Nunit_Console="c:\Program Files (x86)\nunit 2.6.4\bin\nunit-console-x86.exe"
 SET Test_DLL=projects\GKTests\bin\Debug\GKTests.dll
 
+:: Set up our log files.
+SET LogFile=%Temp%\GEDKeeper_NUNIT_Output.log
+SET ErrFile=%Temp%\GEDKeeper_NUNIT_Error.log
+@ECHO Log file:   %LogFile%
+@ECHO Error file: %ErrFile%
+IF EXIST "%LogFile%" Del /f "%LogFile%"
+IF EXIST "%ErrFile%" Del /f "%ErrFile%"
+
+:: This will be the exit code we return to the caller.
+SET Exit_Code=0
+
 :: Check for our dependencies.
 IF NOT EXIST %Nunit_Console% GOTO Err_Nunit
 IF NOT EXIST %Test_DLL% GOTO Err_DLL
 
 :: Run the tests.
-%Nunit_Console% %Test_DLL%
+@ECHO/
+%Nunit_Console% %Test_DLL% /output="%LogFile%" /err="%ErrFile%"
```

Comment:
Overriding the paths to the output files is not very good. This is fraught with side effects. Especially because not all the tools I use are published in the repository ... But this needs to be checked in work. You may not fix this yet.

Reply:
I placed them in the TEMP folder since no log files were created originally--avoiding the need to add entries into the `.gitignore` file.